CMakeLists.txt

```diff
@@ -219,6 +219,13 @@ if(VELOX_BUILD_TESTING OR VELOX_BUILD_TEST_UTILS)
   set(VELOX_ENABLE_PARSE ON)
 endif()
 
+if(VELOX_ENABLE_CUDF)
+  message(STATUS "Building curl from source to satisfy cuDF curl version requirement")
+  set(curl_SOURCE BUNDLED)
+  velox_resolve_dependency(curl)
+  FetchContent_MakeAvailable(curl)
```

Comment:
can be moved to curl.cmake. https://github.com/facebookincubator/velox/pull/14160 will fix this.